### DISCUSSION TITLE
Passkeys store doesn't work

### DISCUSSION THREAD

**Discussion by Lawlietr** [dani-garcia/vaultwarden#4048] [Q&A]
I use that to store the passkey for google and  facebook for my accounnts.
And both soted success, but got the no passkeys when i try to loging to that.
Always said no passkeys was stored.

vaultwarden: 2023.10.0, docker image vaultwarden/server:latest 
browser extensions: 2023.10.1
browser: EDGE, Chrome

thanks

**Comment by leo15dev** [comment]
https://github.com/dani-garcia/vaultwarden/discussions/4034#discussioncomment-7499808

Update your browser extension to 10.2 may solved the issue. Same in https://github.com/dani-garcia/vaultwarden/discussions/4049#discussion-5826548

**Reply by Lawlietr** [reply]
How do i update my extension to 10.2?

I looking the 10.1 it is current version of web extension?
https://chrome.google.com/webstore/detail/bitwarden-free-password-m/nngceckbapebfimnlniiiahkandclblb

thanks

**Reply by Lawlietr** [reply]
I was update to 10.2, but the problem it's get worse hard.
Now the extension never asking where my passkeys i want use for.

**Reply by leo15dev** [reply]
@Lawlietr 

Update

Win 11 23H2 22631.2506
Chrome  118.0.5993.120 
Bitwarden Chrome extension 10.2
Vaultwarden 1.30

Both my Google and Facebook account could login with  the passkeys which stored in the Vaultwarden. There in no issue to select the passkeys in Bitwarden Chrome extension.

**Reply by Lawlietr** [reply]
Nice, but how can i debug or provide log or anything to help me to solve this problem?

My extension will stored passkeys, i can see then in the extension gui, but it really doesn't work on my browser.
Both of EDGE 119.0.2151.44, Chrome 119.0.6045.124.

Thanks

**Comment by TheUntouchable** [comment]
I have the same problem with Firefox 119.0.1 and the extension 10.1 for Google.com. I can create the passkey and also test it on the same website successfully, but when I try to log in, the passkey is not found.

On sites like github, synology and others it works without problems.

EDIT: Btw, you can get the latest plugins here https://github.com/bitwarden/clients/releases, but its not in a usable format.

EDIT2: Also see here: https://github.com/bitwarden/clients/issues/6764

**Reply by Lawlietr** [reply]
Thank you, you let me know i'm not alone.

Good day.

**Comment by pk-1966** [comment]
Same for me using 10.2 on both Chrome and Firefox (using Vaultwarden 1.30.1). 

Windows Hello intercepts the passkey first - Is there a way to change the priority?

**Comment by wioch** [comment]
Same here, passkey for google was created and saved into VW vault correctly, 
When logging back to google, google ask to insert security key, no passkey popup from VW. 
FF: 120.1 
VW client : 10.2
VW server: 1.30

Later I'll check with Edge and Chrome,
